models/repos.js

```diff
@@ -180,82 +184,112 @@ function createModel () {
       })
     })
   }
-}
 
-// creates a wrapper for all dats. Handles stats, and updates choo's internal
-// state whenever a mutation happens
-function createManager (multidat, onupdate) {
-  assert.ok(multidat, 'models/repos: multidat should exist')
-  assert.ok(onupdate, 'models/repos: onupdate should exist')
+  function togglePause (state, data, send, done) {
```

Comment:
needed to change indentation, so this looks messy. however i didn't change anything besides adding the togglePause method and reading from dbPaused